lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -147,11 +161,60 @@ With some familiarity of regrouping and reordering, students are ready to get cr
 - Turn your paper in to your teacher.
 }
 
-We encourage you to review students' submissions, and write a few on the board to discuss as a class. We want to sharpen students' eyes and help them develop the ability to spot instances when they might apply the Associative Property in any context.
+@teacher{We encourage you to review students' submissions, and write a few on the board to discuss as a class. We want to sharpen students' eyes and help them develop the ability to spot instances when they might apply the Associative Property in any context.}
 
 === Synthesize
 
 - How would you describe the relationship between the Associative Property and the Commutative Property? Do you think one is more powerful than the other?
 - How are the Commutative and Associative Properties similar? How are they different?
 
+== Programming Exploration: Associativity @duration{20 minutes}
+
+=== Overview
+
+Students consider whether various functions that we use when coding are associative.
+
+=== Launch
+
+In math, the Associative Property tells us that when adding three numbers or multiplying three numbers, it does not matter whether you start with the first pair or the last. In other words, we can change the groupings and get the same result!
+
+@QandA{
+- Do you think the Associative Property has a role in @proglang?
```

Comment:
I don't understand this question, and it's a yes/no that doesn't help jog discussion. I'd suggest removing it.

Reply:
agreed.